### ISSUE TITLE
Usability and Accessibility issues due to lack of "Back" buttons

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#5731]
The lack of _Back_ Buttons throughout the **Cancel/Return** process prevent keyboard-only users an easy way back to the previous screen(s). This goes against guideline **WCAG 2.0, 2.1.1, Level A, Keyboard**

Provided the **Cancel/Return** process is the opposite of the **Checkout** process where _Back_ buttons are present at every step, navigation logic would have to be similar to _"Uncheckout"_ those items, giving the possibility to go back and correct items, choose a different order if mistaken.

**Usability issue:**
With the current state, if a **mouse user** changes his mind at the last step and hits the ui back button, then it will go back to the cancel/return page (without a ui back button) so by hitting the browser back button it will only get him back to the confirmation page, creating a confusing loop (that could be broken by hitting the browser button **twice**). This scenario would be frustrating for a sighted keyboard-only user, and probably for a blind user will be against guideline **WCAG 2.0, 2.1.2, Level A, No Keyboard Trap** as well.

![Screen Shot 2019-12-05 at 11 24 20 AM](https://user-images.githubusercontent.com/15113219/70550986-a1881e80-1b44-11ea-9569-a74baea08733.png)

![Screen Shot 2019-12-04 at 2 09 11 PM](https://user-images.githubusercontent.com/15113219/70551022-bbc1fc80-1b44-11ea-841d-ab2c485d7919.png)

**Comment by kacperknapik** [comment]
Moving that to Asterix board, because it is not related to a11y for now.

**Comment by ghost** [comment]
As per Bill, We will need a backend with 2005 installed for this. We can postpone this one until a 2005 is ready ( check with Gil)
